include/RAJA/policy/sequential/reduce.hpp

```diff
@@ -158,18 +157,20 @@ class ReduceMinLoc<seq_reduce, T>
   using my_type = ReduceMinLoc<seq_reduce, T>;
 
 public:
+
+  ReduceMinLoc() = delete;
   //
   // Constructor takes default value (default ctor is disabled).
   //
-  explicit ReduceMinLoc(T init_m_val, Index_type init_loc)
+  RAJA_HOST_DEVICE explicit ReduceMinLoc(T init_m_val, Index_type init_loc)
       : m_parent(NULL), m_val(init_m_val), loc(init_loc)
   {
   }
 
   //
   // Copy ctor.
   //
-  ReduceMinLoc(const ReduceMinLoc<seq_reduce, T>& other)
+  RAJA_HOST_DEVICE ReduceMinLoc(const ReduceMinLoc<seq_reduce, T>& other)
```

Comment:
👍 Good point. Though we can also just write `ReduceMinLoc` without the templates